backend/app/api/v1/endpoints/auth.py

```
@@ -0,0 +1,62 @@
+    )
+
+    response = response_handler.success_response(
+        data=login_response["user"], message="User created successfully"
+    )
+    set_auth_cookie(response, login_response["access_token"])
+    return response
+
+
+@router.put("/update-password")
+@auth_required() 
+async def update_password(request: Request, body: PasswordUpdateRequest, user: UserResponse, db: Session):
```

Ruff (0.14.14)

Unused function argument: `request`

(ARG001)